src/vivarium/framework/values.py

```diff
@@ -243,22 +326,22 @@ def _call(self, *args, skip_post_processor=False, **kwargs):
 
         return value
 
-    def __repr__(self):
+    def __repr__(self) -> str:
         return f"_Pipeline({self.name})"
 
 
 class ValuesManager(Manager):
     """Manager for the dynamic value system."""
 
-    def __init__(self):
+    def __init__(self) -> None:
         # Pipelines are lazily initialized by _register_value_producer
-        self._pipelines = defaultdict(Pipeline)
+        self._pipelines: dict[str, Pipeline] = defaultdict(Pipeline)
 
     @property
-    def name(self):
+    def name(self) -> str:
         return "values_manager"
 
-    def setup(self, builder):
+    def setup(self, builder: "Builder") -> None:
```

Comment:
You don't need the quotes here. It would be good to remove quotes in typing everywhere in this file.